helm/prometheus-rules/templates/platform/atlas/alerting-rules/tempo-mixins.rules.yml

```diff
@@ -0,0 +1,370 @@
+          expr: |
+            sum(tempo_runtime_config_last_reload_successful{cluster_type="management_cluster"} == 0) by (cluster_id, installation, pipeline, provider, namespace, job)
+          for: 15m
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoUserConfigurableOverridesReloadFailing
+          annotations:
+            description: '{{`Greater than 5 user-configurable overides reloads failed in the past hour.`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoTenantIndexFailures
+          expr: |
+            sum by (cluster_id, installation, pipeline, provider, namespace) (increase(tempo_overrides_user_configurable_overrides_reload_failed_total{cluster_type="management_cluster"}[1h])) > 5 and
+            sum by (cluster_id, installation, pipeline, provider, namespace) (increase(tempo_overrides_user_configurable_overrides_reload_failed_total{cluster_type="management_cluster"}[5m])) > 0
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoProvisioningTooManyWrites
+          annotations:
+            description: '{{`Ingesters in {{ $labels.cluster_id }}/{{ $labels.namespace }} are receiving more data/second than desired, add more ingesters.`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoProvisioningTooManyWrites
+          expr: |
+            avg by (cluster_id, installation, pipeline, provider, namespace) (rate(tempo_ingester_bytes_received_total{cluster_type="management_cluster", job=~".+/ingester"}[5m])) / 1024 / 1024 > 30
+          for: 15m
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoCompactorsTooManyOutstandingBlocks
+          annotations:
+            description: '{{`There are too many outstanding compaction blocks in {{ $labels.cluster_id }}/{{ $labels.namespace }} for tenant {{ $labels.tenant }}, increase compactor s CPU or add more compactors.`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoCompactorsTooManyOutstandingBlocks
+          expr: |
+            sum by (cluster_id, installation, pipeline, provider, namespace, tenant) (tempodb_compaction_outstanding_blocks{cluster_type="management_cluster", container="compactor"}) / ignoring(tenant) group_left count(tempo_build_info{cluster_type="management_cluster", container="compactor"}) by (cluster_id, installation, pipeline, provider, namespace) > 100
```

Comment:
Can you check if this exists for the compactor? I'm pretty sure the container label is named tempo-compactor (i've seen this on dashboards)